File: leetcode/AIcoding/5.Qwen-3/javascript_solutions/1798-maximum-number-of-consecutive-values-you-can-make.js
function getMaximumConsecutive(coins) {
    coins.sort((a, b) => a - b);
    let max = 0;
    for (let i = 0; i < coins.length; i++) {
        if (coins[i] <= max + 1) {
            max += coins[i];
        } else {
            break;
        }
    }
    return max + 1;
}

File: leetcode/AIcoding/3.DeepSeek-v3/erlang_solutions/1798-maximum-number-of-consecutive-values-you-can-make.erl
-spec get_maximum_consecutive(Coins :: [integer()]) -> integer().
get_maximum_consecutive(Coins) ->
    Sorted = lists:sort(Coins),
    get_maximum_consecutive(Sorted, 1, 0).

get_maximum_consecutive([], Max, _Sum) ->
    Max;
get_maximum_consecutive([H | T], Max, Sum) ->
    if
        H =< Sum + 1 ->
            get_maximum_consecutive(T, Max + H, Sum + H);
        true ->
            Max
    end.